### ISSUE TITLE
Crashing if "User" button is clicked

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#355]
```
What steps will reproduce the problem?
1. In JMM Desktop, change username from "default" to another (alphabetic 
characters only, no numbers, spaces or symbols)
2. Left password blank
3. Launch JMM Desktop, after auto-login, click on "User" button at the bottom 
of JMM.

What is the expected output? What do you see instead?
Expected: usual dailogue box that allows changing which user is logged in.
Saw: JMM Desktop crashes.

What version of the product are you using? On what operating system?
About still says 3.1.23.0 after running update (to 3.1.24.0), Win8 x64

Please provide any additional information below.
Ran updates on both JMM Desktop when they were both 3.1.23.0, so expected both 
to advance to 3.1.24.0, but the version has only changed on the Server's 
"About". Both say they're on the latest version though. All launch with 
elevated privileges. 
Error dialogue "problem details" upon crash:
======
Problem signature:
  Problem Event Name:   CLR20r3
  Problem Signature 01: jmmdesktop.exe
  Problem Signature 02: 3.1.23.0
  Problem Signature 03: 50b5a489
  Problem Signature 04: System
  Problem Signature 05: 4.0.30319.17929
  Problem Signature 06: 4ffa5c88
  Problem Signature 07: 2fbb
  Problem Signature 08: 76
  Problem Signature 09: DI4EQZGVCWKDLTHKQTZWPXE3YRFKN1I2
  OS Version:   6.2.9200.2.0.0.256.48
  Locale ID:    12297
  Additional Information 1: aa00
  Additional Information 2: aa005761d7f24b5612ba5ab1f5fe4e07
  Additional Information 3: 9798
  Additional Information 4: 97989cca3e4728823032c03d5b62eadb
======
```

Original issue reported on code.google.com by `inferk...@gmail.com` on 10 Dec 2012 at 11:49

**Comment by GoogleCodeExporter** [comment]
```
Can you check your Windows Event Viewer (Application), to see if there is 
anything more detailed?
```

Original comment by `werndly...@gmail.com` on 10 Dec 2012 at 11:13

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Alright, I recreated the error with Event Viewer opened now.
A set of 3 errors in a row show up in Event Viewer.
I've listed the errors in the attached text file (bottom event to top event).

Then at the top of the Event viewer list is just the Windows Error Reporting 
event.
I hope that helps.
```

Original comment by `inferk...@gmail.com` on 11 Dec 2012 at 12:46

Attachments:
- [JMM Desktop Error.txt](https://storage.googleapis.com/google-code-attachments/jmm/issue-355/comment-3/JMM Desktop Error.txt)

**Comment by GoogleCodeExporter** [comment]
```
Doh, I left some test code in there. Will release a fix in next couple of hours
```

Original comment by `werndly...@gmail.com` on 11 Dec 2012 at 8:52

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.25
```

Original comment by `werndly...@gmail.com` on 11 Dec 2012 at 9:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `werndly...@gmail.com` on 11 Dec 2012 at 9:28
- Changed state: **Fixed**
